Main.cpp

```diff
@@ -140,20 +140,22 @@ enum StarSize {
 
 struct Star {
     // Bitmap representation
-    BITMAP *m_pBitmap;
+    BITMAP *m_pBitmap = nullptr;
     // Center location on screen
-    Vector m_Pos;
-    // Bitmap offset
-//    int m_Offset;
+	std::array<int, 2> m_Pos = { 0 };
```

Comment:
Seems odd to have spaces in a list initializer like this. If you were putting in both values you'd probably do `{1, 2}` not `{ 1, 2 }`, right.

Reply:
That's my auto-formatting :-\.
Posted something formatting-related in the discord.
Anyway, removed this in favor of X and Y variables.